src/modules/Cart/CartBody/component.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import { Typography, Button } from 'antd'
+import { CloseCircleOutlined } from '@ant-design/icons'
+import useContainer from '../hook'
+import styles from '../styles.module.scss'
+
+const { Text } = Typography
+
+const CartBody: React.FC = () => {
+  const { cart, handleDeleteProduct } = useContainer()
+
+  return (
+    <ul>
+      {cart.map(product => {
+        return (
+          <li key={product.id} className={styles.liProductPosition}>
+            <div className={styles.cartInfoBlock}>
+              <div>
+                <Text className={styles.productName}>{product.name}</Text>
+                <div className={styles.priceQuantity}>
```

Comment:
Add space between lines on the same level

Reply:
done